new-site/src/components/Hero.scss

```diff
@@ -0,0 +1,149 @@
+@import "./utils.scss";
+
+.hero-container {
+  --koros-height: 145px;
+  --hero-height: 540px;
+  --hero-small-left: calc((100vw - 100%) / 2);
+  --hero-large-content-left: calc((100vw + var(--spacing-s) - var(--container-width-xl)) / 2);
+
+  background-color: var(--color-coat-of-arms);
+  box-sizing: border-box;
+  position: relative;
+  left: calc(var(--hero-small-left) * -1);
+  width: 100vw;
+
+  @include x-large-and-above-screen {
+    left: calc(var(--hero-large-content-left) * -1);
+    padding-left: var(--spacing-s);
+    z-index: 1;
+    width: 100vw;
+  }
+}
+
+.hero-bg-image {
+  aspect-ratio: 16 / 9;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  margin-top: calc(var(--koros-height) / -2);
+
+  @include x-large-and-above-screen {
+    aspect-ratio: unset;
+    background-size: cover;
+    background-position: center;
+    height: 100%;
+    margin-top: 0;
+    left: var(--hero-large-content-left);
+    right: 0;
+    position: absolute;
+    top: 0;
+  }
+}
+
+.hero-wrapper {
+  margin: 0 auto;
+  max-width: var(--container-width-xl);
+}
+
+.hero {
+  color: var(--color-white);
+  margin: 0 auto;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
```

Comment:
The margin and width rules did not seem to have an effect when tested without them.

Reply:
- [ ] Check if margin and width can be omitted